packages/webpack-inline-constant-exports-plugin/index.js

```diff
@@ -148,9 +149,29 @@ class InlineConstantExportsPlugin {
 							 *
 							 * When inlining a constant, we remove the ImportSpecifierDependency and replace it
 							 * with a ConstDependency that inserts the inlined code verbatim.
+							 *
+							 * ---
+							 *
+							 * Webpack 5 keep tracks of re-exported dependencies:
+							 *   //const.js
+							 *   export const FOO=1;
+							 *
+							 *   //api.js
+							 *   export * as api from 'const';
+							 *
+							 *   //index.js
+							 *   import { api } from 'api';
+							 *   console.log( api.FOO );
+							 *
+							 * But this plugin does not support them. If a dependency has multiple IDs it means it is a
+							 * re-exported dep (eg: `api.FOO`), so we skip it.
```

Comment:
Is this lack of support for reexports a new thing in webpack 5, or are you merely documenting something that was already true before, in webpack 4?

Reply:
Our plugin didn't support inlining re-exports in webpack 4, but webpack 4 API made it really hard to do support them. With the APIs in Webpack 5 it should be easier to support re-exports (at least in theory), but I haven't implemented it yet.

This comment is mostly to explain why we do `const depId = dep.ids[ 0 ];` instead of looping over all available `dep.ids`.

Reply:
Thanks for the clarification. :+1: We use the plugin just with one module (Redux action types), and its exports are never reexported. So adding the reexport support is very low-priority.